CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+## 2.3.0 (Mar 29, 2016)
+### general
+ - Dynamic Config: Added a new feature to track config file for changes and restart the 
+   pipeline (same process) with updated config changes. This feature can be enabled in two 
+   ways: Passing a CLI long-form option `--auto-reload` or with short-form `-r`. Another 
+   option: `--reload-interval <seconds>` controls how often LS should check the config files 
+   for changes. Alternatively, if you don't start with the CLI option, you can send SIGHUP 
+   or `kill -1` to signal to LS to reload the config file and restart the pipeline ([#4513](https://github.com/elastic/logstash/issues/4513)).
+ - Added support to evaluate environment variables inside LS Config. You can also specify a 
+   default if the variable is not defined. The syntax is `${myVar:default}` ([#3944](https://github.com/elastic/logstash/issues/3944)).
+ - Improved throughput performance across the board (up by 2x in some configs) by implementing Event 
+   representation in Java. Event is the main object which encapsulates data as it flows through 
+   Logstash and provides APIs for the plugins to perform processing. This change also enables 
+   faster serialization for future persistence work ([4191](https://github.com/elastic/logstash/issues/4191)).
+ - Add ability to configure custom GC log file using `$LS_LOG_DIR`.
+ - Deprecate `bin/plugin` in favor of `bin/logstash-plugin`. In the next major version `bin/plugin` will 
+   be removed to prevent `PATH` being polluted when other components of Elastic stack is installed on 
+   the same instance ([#4891](https://github.com/elastic/logstash/pull/4891))
+ - Fixed a bug where new pipeline might break plugins by calling the `register` method twice causing 
```

Comment:
Suggest removing "causing undesired behavior" because the language is too vague to add meaning here.  Maybe say: "Fixed a bug where new pipelines could break plugins by calling the `register` method twice"